package.json

```diff
@@ -153,6 +153,7 @@
     "compare-versions": "^4.1.1",
     "connected-react-router": "^6.6.1",
     "crypto-browserify": "^3.12.0",
+    "date-fns": "^3.6.0",
```

Comment:
We'll also need to include this dependency in https://github.com/eosdis-nasa/earthdata-pub-dashboard/blob/develop/dependency_list.txt